src/snowflake/cli/api/project/definition_conversion.py

```diff
@@ -334,12 +324,11 @@ def _convert_package_script_files(package_scripts: list[str]):
             package["distribution"] = native_app.package.distribution
         package_meta = _make_meta(native_app.package)
         if native_app.package.scripts:
-            converted_post_deploy_hooks = _convert_package_script_files(
-                native_app.package.scripts
-            )
-            package_meta["post_deploy"] = (
-                package_meta.get("post_deploy", []) + converted_post_deploy_hooks
-            )
+            # Package scripts are not supported in PDFv2 but we
+            # don't convert them here, conversion is deferred until
+            # the final v2 Pydantic model is available
+            # (see _convert_templates_in_files())
+            pass
```

Comment:
left this in instead of removing the `if` as a marker that this case is indeed being considered but consciously ignored